src/main/java/org/folio/circulation/resources/RegularRenewalStrategy.java

```diff
@@ -107,6 +108,11 @@
           loan.getItemId()));
         return failedValidation(errors);
       }
+      if (loan.hasItemWithStatus(ItemStatus.CLAIMED_RETURNED)) {
+        errors.add(itemByIdValidationError(CLAIMED_RETURNED_RENEWED_ERROR,
+          loan.getItemId()));
+        return failedValidation(errors);
+      }
```

Comment:
Personally, I might include some whitespace on either side of this, to separate it from the surrounding code